src/test/kotlin/io/github/mojira/arisa/modules/FutureVersionModuleTest.kt

```diff
@@ -276,8 +282,28 @@ class FutureVersionModuleTest : StringSpec({
         result.shouldBeRight(ModuleResponse)
     }
 
+    "should not resolve if there is another version" {
+        var isResolved = false
+
+        val module = FutureVersionModule("message", "panel")
+        val issue = mockIssue(
+            created = FIVE_SECONDS_AGO,
+            affectedVersions = listOf(FUTURE_VERSION, RELEASED_VERSION),
+            changeLog = listOf(ADD_FUTURE_VERSION),
+            project = mockProject(
+                versions = listOf(RELEASED_VERSION)
+            ),
+            resolveAsAwaitingResponse = { isResolved = true }
```

Comment:
Looks like we could also test the comment here

Reply:
Probably should also add another test for testingthat the correct comment is posted in the other case